### PR TITLE
Service required resources adaptation

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: aostypes/aostypes.go [hunk 1]
```diff
@@ -201,6 +201,13 @@ type ResourceRatiosInfo struct {
 	Storage *float64 `json:"storage"`
 }
 
+// RequestedServiceResources resources required for a service(in absolute values: dmips, bytes).
```
Comment (al1img) [inline aostypes/aostypes.go:204]: space before (

**Review by MykolaSolyanko — APPROVED** [review]

**Review by mlohvynenko — APPROVED** [review]

**Review by mlohvynenko — APPROVED** [review]

File: aostypes/aostypes.go (hunk 1 above)
Comment (mykola-kobets-epam) [reply aostypes/aostypes.go]: fixed

File: aostypes/aostypes.go [hunk 2]
```diff
@@ -201,6 +201,13 @@ type ResourceRatiosInfo struct {
 	Storage *float64 `json:"storage"`
 }
 
+// RequestedServiceResources resources required for a service (in absolute values: dmips, bytes).
+type RequestedServiceResources struct {
```
Comment (al1img) [inline aostypes/aostypes.go:205]: Rename to `RequestedResources`

Comment (mykola-kobets-epam) [reply aostypes/aostypes.go]: done

**Review by mlohvynenko — APPROVED** [review]

**Review by al1img — APPROVED** [review]

**Pull request merged**